Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue8781.xaml.cs

```diff
@@ -25,17 +26,28 @@ public Issue8781()
 		{
 #if APP
 			InitializeComponent();
+			BindingContext = this;
 #endif
 		}
 
-		async void OnIncorrectAnswerInvoked(object sender, EventArgs e)
-		{
-			await DisplayAlert("Incorrect!", "Try again.", "OK");
-		}
+		public ICommand CheckAnswerCommand => new Command<string>(OnCheckAnswer);
 
-		async void OnCorrectAnswerInvoked(object sender, EventArgs e)
+		async void OnCheckAnswer(string parameter)
 		{
-			await DisplayAlert("Correct!", "The answer is 2.", "OK");
+			if (string.IsNullOrEmpty(parameter))
```

Comment:
```suggestion
			#if APP
			if (string.IsNullOrEmpty(parameter))
```

Reply:
Right. Thanks for the feedback. Updated this repro sample to fix the build.